docs/_data/tools_list.yml

```diff
@@ -284,3 +284,7 @@
     - name: GitHub
       url: https://github.com/kit-data-manager/NovaCrate
     
+- name: ROCrate Profile Crate Creator
+  description: Polyglot library for creation of RO-Crate Profile Crates (Available in Python, Typescript and FSharp)
+  status: beta
+  url: https://github.com/nfdi4plants/ROCratePCC
```

Comment:
Could you add subitems with the links to the package on PyPI/npm/etc? example with subitems:

```
- name: Describo
  description: An intuitive and extensible tool for working with RO-Crates. Transform, Discover, Explore, Describe.
  status: production
  level:
  url: https://describo.github.io
  subitems:
    - name: Component - embed the Describo RO-Crate engine in your application
      url: https://describo.github.io/docs/component/introduction.html
    - name: Profiles - adapt describo to your needs
      url: https://describo.github.io/docs/profiles/introduction.html
```